src/libwalletqt/TransactionHistory.cpp

```diff
@@ -80,7 +80,7 @@ void TransactionHistory::refresh(quint32 accountIndex)
 
         m_pimpl->refresh();
         for (const auto i : m_pimpl->getAll()) {
```

Comment:
I think you need to add `&` to `const auto`, or it will make a copy of each transaction in this loop.

Reply:
Did not notice this, good point - changed.